ti/instrumentation/utils.go

```diff
@@ -264,6 +267,151 @@ func getChangedFiles(ctx context.Context, workspace string, log *logrus.Logger,
 	return res, nil
 }
 
+// addBazelFilesToChangedFiles takes a list of files and removes bazel build files and adds java files listed in target src globs
+func addBazelFilesToChangedFiles(ctx context.Context, workspace string, log *logrus.Logger, res []ti.File, bazelFileCount string) ([]ti.File, []string, error) {
```

Comment:
We should name the parameters such that they are self explanatory for the most part. For example "res" is not.
Also consider `bazelFileCount` -> `bazelFileCountThreshold`